File: app/src/main/java/com/example/a18/path/patcheffect/PathEffectActivity.java
package com.example.a18.path.patcheffect;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.a18.path.R;

public class PathEffectActivity extends AppCompatActivity {

    private PathEffectView pathEffectView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_path_effect);

        pathEffectView = findViewById(R.id.pev);
        pathEffectView.post(new Runnable() {
            @Override
            public void run() {
pathEffectView.start();
            }
        });

    }
}

File: m_patheffect/src/main/java/pdog18/com/module_patheffect/PathEffectActivity.kt
package pdog18.com.module_patheffect

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class PathEffectActivity : AppCompatActivity() {

    private var pathEffectView: PathEffectView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_path_effect)

        pathEffectView = findViewById<PathEffectView>(R.id.pev)
        pathEffectView!!.post(Runnable { pathEffectView!!.start() })

    }
}
